PanoptesNetClient/PanoptesNetClient/ApiClient.cs

```diff
@@ -7,54 +7,49 @@
 namespace PanoptesNetClient
 {
     public class ApiClient
-    {
-        static HttpClient client = new HttpClient();
-        private static ApiClient instance;
+    { 
+        private static HttpClient Client;
 
-        private ApiClient()
+        private void ConfigClient()
         {
-            client.BaseAddress = new Uri(Config.Host);
-            client.DefaultRequestHeaders.Accept.Clear();
-            client.DefaultRequestHeaders.Accept.Add(
-                new MediaTypeWithQualityHeaderValue("application/json"));
-            client.DefaultRequestHeaders.Add("Accept", "application/vnd.api+json; version=1");
- 
-            RunAsync().GetAwaiter().GetResult();
+            Client.BaseAddress = new Uri(Config.Host);
+            Client.DefaultRequestHeaders.Accept.Clear();
+            Client.DefaultRequestHeaders.Accept.Add(
+              new MediaTypeWithQualityHeaderValue("application/json"));
+            Client.DefaultRequestHeaders.Add("Accept", "application/vnd.api+json; version=1");
         }
 
-        public static ApiClient Instance
+        public ApiClient(HttpClient client = null)
         {
-            get
+            if (client == null)
             {
-                if (instance == null)
-                {
-                    instance = new ApiClient();
-                }
-                return instance;
+                client = new HttpClient();
             }
+            Client = client;
+            ConfigClient();
         }
 
-        private async Task RunAsync()
+        public IRequest Type(string resource)
         {
-            try
-            {
-                await GetProject();
-            }
-            catch (Exception e)
-            {
-                Console.WriteLine(e.Message);
-            }
+            IRequest request = new Request(resource);
+            return request;
         }
 
-        private async Task<JObject> GetProject()
+        public static async Task<JObject> GetAsync(IRequest request)
```

Comment:
Ah yes. I think you're correct and `static` can be removed since we are instantiating this now. This logic might be leftover from when I was thinking of it as a singleton.